File: Dessarrollo_WEB_cliente (DWC)/2_Evaluacion/Angular/ProyectoDWC/src/app/Model/game.ts
import { Character } from "./character";
import { Director } from "./director";

export interface Game {
    id: number;
    gameCode: string;
    title: string;
}

export interface GameDetail {
    id: number;
    gameCode: string;
    title: string;
    releaseYear: string;
    description: string;
    director: Director;
    characterGames: Character[];
}

File: Dessarrollo_WEB_servidor (DWS)/2_Evaluacion/ProyectoDWS/src/main/java/com/fpmislata/exam/domain/model/Game.java
package com.fpmislata.exam.domain.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@RequiredArgsConstructor
public class Game {
    private Integer id;
    private String gameCode;
    private String title;
    private String releaseYear;
    private String description;
    private Director director;
    private List<CharacterGame> characterGames;
}
